app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,60 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_sign_up_params, only: [:create]
+  # before_action :configure_account_update_params, only: [:update]
+
+  # GET /resource/sign_up
+  # def new
+  #   super
+  # end
+
+  # POST /resource
+  # def create
+  #   super
+  # end
+
+  # GET /resource/edit
+  # def edit
+  #   super
+  # end
+
+  # PUT /resource
+  # def update
+  #   super
+  # end
+
+  # DELETE /resource
+  # def destroy
+  #   super
+  # end
+
+  # GET /resource/cancel
```

Comment:
Hi @BenMukebo. I have a question here. For example, this code was automatically generated by a command and could be useful in the future if I decide to implement features requiring it. Would not be better if I don't delete these comments?

Reply:
Hello @CindyDorantes, I understand what you most of the time when I'm working I use to make one commit which the last one when I deleted all the commented line so that i can I wanna get reference to them, i can easy checkout in back on commit.
- Because this is one of the Microverse role to keep the code clean. we're oblige to remove all the commented line apart form those in the config folder(initialize, ...)